resources/inspectionDescriptions/PreferenceXmlInspections.html

```diff
@@ -0,0 +1,18 @@
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<html>
+<body>
+<p>
+  Validates classes and interfaces for  attributes "for" and the "type" for the "preference" tag
+</p>
+<p>This inspection checks the "for" and "type" attribute of the &ltpreference/&gt; tag for:</p>
+<ul>
+  <li>the correct path to the class or interface</li>
+  <li>existence of attribute values</li>
+</ul>
+</body>
+</html>
```

Comment:
@Iamwade, please add an empty line in the and of the file.

Reply:
Fixed.

Reply:
Thank you!